### ISSUE TITLE
Developer portal link in git or arc commit messages?

### ISSUE THREAD

**Issue by tomocafe** [getsolus/help-center-docs#242]
The article "Submitting a Package" (section "Commit") seems to indicate you should put tracker tasks / etc. in the git commit message.

> If you want to link this patch to an issue on the Developer portal, simply mention it in your commit message: The inclusion of <somepackage> fixes T1234

But these should go in the arc diff message instead.

This one always trips me up, so I end up having to switch around my git and arc messages.

**Comment by TraceyC77** [comment]
@tomocafe , we are in the process of updating the packaging docs after migrating to GitHub. To my understanding, the link to another issue would be in the commit message now.

The packaging docs PR is https://github.com/getsolus/help-center-docs/pull/373. Once that is merged, would you mind taking another look at this page? It's now at https://help.getsol.us/docs/packaging/submitting-a-package

**Comment by TraceyC77** [comment]
Closing, since GitHub handles commits differently than arc diff did. Please open a new issue if you find problems with the current documentation.
